### PR TITLE
Added 'global' comments to remove eslint 'no-undef' errors + minor bug fixes

### PR DESCRIPTION
**Pull request by cgorringe**
Fixing some issues reported by eslint. Also tweaked README.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -32,20 +32,22 @@ If you want to install the latest version, follow the steps below.
...
-  - Click “This Firefox”
-  - Click “Load Temporary Add-on”
-  - Open your extension’s directory and select any file inside the extension.
+### Firefox
 
-This is a "WebExtension". See https://developer.chrome.com/extensions for more information.
+- Open Firefox and navigate to `about:debugging` in the browser.
+- Click **This Firefox**.
+- Click **Load Temporary Add-on...**
+- Open the `wayback-machine-chrome` directory and select any file.
```
Comment (tikhsuP) [inline README.md:51]: We can add this line:

See [develop/temporary-installation-in-firefox](https://extensionworkshop.com/documentation/develop/temporary-installation-in-firefox/) for more information on Firefox temporary add-ons.

Comment (cgorringe) [reply README.md:51]: Thanks, I'll add it on a future PR!

File: scripts/tagcloud.js [hunk 2]
```diff
@@ -48,7 +54,7 @@ function get_tags (url) {
           }
           return acc
         }, []).sort()
-        var result = new Array()
+        var result = []
```
Comment (anishsarangi) [inline scripts/tagcloud.js:57]: It doesn't make any difference I think in this case. !!

Comment (cgorringe) [reply scripts/tagcloud.js:57]: Yeah it's no different. Just eslint complains.  ¯\_(ツ)_/¯

**Pull request merged**